modules/govuk_ci/manifests/job.pp

```diff
@@ -0,0 +1,40 @@
+# == Define: govuk_ci::job
+#
+# Define to manage a job for GOV.UK CI
+#
+# === Parameters
+#
+# [*app*]
+# Application name (Default: title)
+#
+# [*repository*]
+# Repository name, it needs to match the Github repository name (Default: title)
+#
+define govuk_ci::job (
+  $app = $title,
+  $repository = $title,
+) {
+
+  $job_config_directory = '/var/lib/jenkins/jobs'
+  $application_directory = "${job_config_directory}/${app}"
+
+  file { '/var/lib/jenkins/jobs':
```

Comment:
Might be worth turning this in to an array of this and the following directory as the properties are all the same.